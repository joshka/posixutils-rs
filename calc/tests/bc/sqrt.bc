a = sqrt(0)
scale(a)
a

a = sqrt(1)
scale(a)
a

a = sqrt(2)
scale(a)
a

scale = 10
a = sqrt(2)
scale(a)
a

scale = 0
a = sqrt(2.00000)
scale(a)
a

a = sqrt(4)
scale(a)
a

a = sqrt(0.25)
scale(a)
a
quit
