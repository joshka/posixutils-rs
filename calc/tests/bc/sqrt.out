0
0
0
1
0
1
10
1.4142135623
5
1.41421
0
2
2
0.50
